SeaPublicWebsite.BusinessLogic/Models/Enums/AccessibleLoft.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GovUkDesignSystem.Attributes;
+
+namespace SeaPublicWebsite.BusinessLogic.Models.Enums
+{
+    public enum AccessibleLoft
```

Comment:
I think `LoftAccess` is  clearer for naming than `AccessibleLoft` here and in the view model and page names

Reply:
I agree